Tracker/tracker.lua

```diff
@@ -8567,6 +8653,25 @@ function tracker:updateNames()
   end
 end
 
+function tracker:getNoteNames()
+  -- track_num as returned here is one-indexed, but as used by GetTrackMIDINoteName, it's zero-indexed (christ)

```

Comment:
Add an ```if tracker.track then``` to avoid problems when no midi item is selected when the tracker is opened.